cmd/promote/promote.go

```diff
@@ -0,0 +1,122 @@
+package promote
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"code.storageos.net/storageos/c2-cli/cmd/argwrappers"
+	"code.storageos.net/storageos/c2-cli/cmd/interfaces"
+	"code.storageos.net/storageos/c2-cli/cmd/runwrappers"
+	"code.storageos.net/storageos/c2-cli/pkg/health"
+	"code.storageos.net/storageos/c2-cli/pkg/id"
+)
+
+type promoteCommand struct {
+	config interfaces.ConfigProvider
+	client interfaces.Client
+
+	cordonTargetNode bool
+}
+
+// NewCommand configures the promote command
+func NewCommand(client interfaces.Client, config interfaces.ConfigProvider) *cobra.Command {
+	c := &promoteCommand{
+		config: config,
+		client: client,
+	}
+
+	command := &cobra.Command{
+		Hidden: true,
+		Use:    "promote",
+		Short:  "promote promotes a volume's replica deployment to a primary",
```

Comment:
This is not a go comment so we don't need the first "promote ". Removing it keeps the context and makes it easier to read